phpunit/functional/NotificationTargetTest.php

```diff
@@ -638,4 +640,51 @@ public function testGetTargetsWithExclusions()
         $target = reset($targets);
         $this->assertEquals(getItemByTypeName('User', TU_USER, true), $target['users_id']);
     }
+
+    public function testAdminTargetInstall()
+    {
+        $this->login();
+
+        $notification = new \Notification();
+        $notifications = $notification->find();
+
+        $tables = require(GLPI_ROOT . '/install/empty_data.php');
+        foreach ($notifications as $notification) {
+            $notification_target = NotificationTarget::getInstanceByType($notification['itemtype'], $notification['event']);
+            $notification_target->addNotificationTargets(0);
+            $filtered = array_filter(
+                $tables["glpi_notificationtargets"],
```

Comment:
You should fetch the targets from the `glpi_notificationtargets` table contents (`$DB->request(['FROM' => \NotificationTarget::getTable(), 'WHERE' => ['notifications_id' => $notification['id']]])`), not from the `empty_data.php` that may have its structure refactored in the future.